microhapulator/marker.py

```diff
@@ -115,24 +129,6 @@ def validate(self, refrids=None, symmetric=False):
                 message = f"reference sequences with no marker definition: {extra}"
                 raise ValueError(message)
 
-    @property
-    def has_chrom_offsets(self):
-        positives, negatives = 0, 0
-        for locus, marker in self:
```

Comment:
Removed this method and all references to it. This will also clean up report generation.